cypress/integration/lists_test/account_list_test.js

```diff
@@ -0,0 +1,158 @@
+            cy.contains("Save").click();
+        })
+        cy.get(".logon.dropdown.success").click().within(() =>{
+            cy.contains("Logout").click();
+        });
+        cy.get(".logon.dropdown.warning").click().should("exist");
+        cy.get('.lists-item').its("length").should("be.gt",0);
+    })
+
+    it("can add tags to a list",function(){
+        cy.createGeneList("SODB, GBP, GST, CDPK1");
+        cy.url().should("include","/lists");
+        cy.get('.lists-item').its("length").should('be.gt',0);
+
+        cy.get(".lists-item").first().within(() => {
+            cy.get("button").find('svg[class="icon icon-list-more"]')
+            .should("be.visible")
+            .click();
+
+            cy.get("button").find('svg[class="icon icon-list-edit"]')
+            .should("be.visible")
+            .first()
+            .click();
+        });
+
+        cy.get(".modal-body").within(()=>{
+            cy.contains("Tags").parent().find("input").type("Major genes{enter}",{delay:100});
+        })
+        cy.get('.modal-footer').within(()=>{
+            cy.contains("Save").click();
+        })
+        cy.get(".lists-table").within(()=>{
+            cy.get(".tag").should("contain","Major genes");
+        })
+    })
+
+    it("can filter lists by tags", function(){
+        cy.contains("Lists").click();
+        cy.url().should("include","/lists");
+        cy.get('.lists-item').its("length").should('be.gt',0);
+
+        cy.get('.lists-headers').within(() => {
+            cy.contains("Tags").siblings(".dropdown").click();
+            cy.get("li").contains("Major genes").click();
+        })
+        cy.get('.lists-item').its("length").should('eq',1);
+    })
+
+    it("can move list(s) to folder",function(){
+        cy.contains("Lists").click();
+        cy.url().should("include","/lists");
+        cy.get('.lists-item').its("length").should('be.gt',0);
+
+        cy.get('.lists-headers').within(() => {
+            cy.get('[type="checkbox"]').check();
+        })
+        cy.get('.bottom-controls').within(() => {
+            cy.contains("Move all").click();
+        })
+        cy.get(".css-1wy0on6").click().type("List of common genes{enter}",{delay:100},{force:true});
+        cy.get(".modal-footer").within(() => {
+            cy.contains("Move list(s)").click();
+        })
+    })
+
+    it("can copy list(s) to folder", function(){
+        cy.contains("Lists").click();
+        cy.url().should("include","/lists");
+        cy.get('.lists-item').its("length").should('be.gt',0);
+
+        cy.get('.lists-headers').within(() => {
+            cy.get('[type="checkbox"]').check();
+        })
+        cy.get('.bottom-controls').within(() => {
+            cy.contains("Copy all").click();
+        })
+        cy.get(".css-1wy0on6").click().type("List 2{enter}",{delay:100},{force:true});
+        cy.get(".modal-footer").within(() => {
+            cy.contains("Copy list(s)").click();
+        })
+    })
```

Comment:
Can you add assertions to the end of these two tests, checking that the folder with that name is visible?